app/controllers/users_controller.rb

```diff
@@ -1,4 +1,8 @@
 class UsersController < ApplicationController
+
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
```

Comment:
Interesting, this is one way to solve the "error" (more of a nice reminder to us developers) coming from `verify_authorized` in the `application_controller.rb`.

However, it makes the policy we set up in user_policy ineffective. Instead, try calling `authorize` on `@user`, similarly to what you did in the `photos_controller`